test/integration/load/inspec/default_spec.rb

```diff
@@ -37,12 +42,6 @@
   describe file('/sys/module/btusb/parameters/reset') do
     its('content') { should match 'Y' }
   end
-
-  # Shouldn't be any init module script
-  describe file(module_path(:init, 7, 'btusb')) do
-    it { should_not  exist }
-  end
-
```

Comment:
Those test should not be remove since we when to test that the 'load' action also configure the module.